ci-operator/jobs/openshift/odo/openshift-odo-main-presubmits.yaml

```diff
@@ -0,0 +1,369 @@
+presubmits:
+  openshift/odo:
+  - agent: kubernetes
+    always_run: true
+    branches:
+    - main
+    cluster: build01
+    context: ci/prow/psi-kubernetes-integration-e2e
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      ci-operator.openshift.io/variant: psi
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-odo-main-psi-kubernetes-integration-e2e
+    optional: true
+    rerun_command: /test psi-kubernetes-integration-e2e
```

Comment:
We need to add `run_if_changed` field for all the prow jobs running on pr, no? https://github.com/openshift/release/blob/master/ci-operator/jobs/openshift/odo/openshift-odo-master-presubmits.yaml#L19

Reply:
done